components/layout/text_run.rs

```diff
@@ -210,6 +210,7 @@ impl<'a> TextRun {
                     ByteIndex(range.start as isize),
                     ByteIndex(range.len() as isize),
                 ),
+                text: text.to_string(),
```

Comment:
This added code to the old layout is due to the modification made to `fonts/glyph.rs` where I need to pass the original text inside a `TextRun` so I can count the actual character count.